en_US/deploy/install.md

```diff
@@ -22,98 +22,123 @@ Purge database directory `/var/lib/emqx/data/mnesia` before starting it with a d
 :::
 
 {% emqxce %}
-1.  Download EMQX package [emqx.com](https://www.emqx.com/en/downloads/try?product=broker) or [Github](https://github.com/emqx/emqx/releases)
+
+1. Download EMQX package [emqx.com](https://www.emqx.com/en/downloads/try?product=broker) or [Github](https://github.com/emqx/emqx/releases)
 {% endemqxce %}
+
 {% emqxee %}
-1.  Download EMQX package [emqx.com](https://www.emqx.com/en/downloads/try?product=enterprise) or [Github](https://github.com/emqx/emqx/releases)
+
+1. Download EMQX package [emqx.com](https://www.emqx.com/en/downloads/try?product=enterprise) or [Github](https://github.com/emqx/emqx/releases)
 {% endemqxee %}
 
 2. Install EMQX Broker:
 
     + RPM package (RedHat/CentOS/RockyLinux/AmazonLinux):
 
       ```shell
-      $ sudo yum install path/to/emqx-full-package-name.rpm
+      sudo yum install path/to/emqx-full-package-name.rpm
       ```
 
     + DEB package (Ubuntu/Debian/Raspbian):
 
-      ```bash
-      $ sudo apt install path/to/emqx-full-package-name.deb
+      ```shell
+      sudo apt install path/to/emqx-full-package-name.deb
       ```
 
 3. Start EMQX and check status
 
       - quick start
 
-        ```bash
-        $ sudo su - emqx
-        $ emqx start
-        $ emqx_ctl status
+        ```shell
+        sudo su - emqx
+        emqx start
+        emqx_ctl status
         ```
 
       - systemctl
 
-        ```bash
-        $ sudo systemctl start emqx
+        ```shell
+        sudo systemctl start emqx
         ```
 
 4.  Stop EMQX Broker
 
-    ```bash
+    ```shell
     $ emqx stop
     ```
 
 5.  Uninstall EMQX Broker
 
     + DEB:
 
-      ```bash
-      $ sudo apt remove --purge emqx
+      ```shell
+      sudo apt remove --purge emqx
       ```
 
     + RPM:
 
-      ```bash
-      $ sudo yum remove emqx
+      ```shell
+      sudo yum remove emqx
       ```
 
-## tgz
+## tgz (Linux, macOS, Windows)
 
 ::: warning
 tar.gz packages are released for quick testing and hot-beam upgrades.
 Do NOT install tar.gz packages for production unless
 you know how to manually resolve all the runtime dependencies.
 :::
 
+::: warning
+The EMQX digital signature work on macOS has not been completed, and the installation and startup of the tgz package may be blocked by [Gatekeeper](https://support.apple.com/zh-cn/guide/security/sec5599b66df/web).
+
+When you encounter a prompt like **“erl” cannot be opened because the developer cannot be verified** or **“erlang_jq_port” cannot be opened because the developer cannot be verified** at startup, handle it as follows:
```

Comment:
```suggestion
When you encounter a prompt like **“erl” cannot be opened because the developer cannot be verified** or **“erlang_jq_port” cannot be opened because the developer cannot be verified** at startup, solve this as follows:
```